integration_tests/dbt_project.yml

```
@@ -1,20 +1,22 @@
 name: 'shopify_integration_tests'
-version: '0.5.2'
+version: '0.7.0'
 profile: 'integration_tests'
 config-version: 2
 
 
 vars:
   shopify_source:
-    customer_source:          "{{ ref('shopify_customer_data') }}"
-    order_line_refund_source: "{{ ref('shopify_order_line_refund_data') }}"
-    order_line_source:        "{{ ref('shopify_order_line_data') }}"
-    order_source:             "{{ ref('shopify_order_data') }}"
-    product_source:           "{{ ref('shopify_product_data') }}"
-    order_adjustment_source:  "{{ ref('shopify_order_adjustment_data') }}"
-    product_variant_source:   "{{ ref('shopify_product_variant_data') }}"
-    refund_source:            "{{ ref('shopify_refund_data') }}"
-    transaction_source:       "{{ ref('shopify_transaction_data') }}"
+    shopify_customer_source_identifier:          "shopify_customer_data"
+    shopify_order_line_refund_source_identifier: "shopify_order_line_refund_data"
+    shopify_order_line_source_identifier:        "shopify_order_line_data"
+    shopify_order_source_identifier:             "shopify_order_data"
```

Pretty sure this is the only change needed for circle to pass
```suggestion
    order_source:                         "{{ ref('shopify_order_data') }}"
```